### ISSUE TITLE
ENOENT: no such file or directory, open node_modules\@angular\core\esm\src\animation\my-component-tpl.html

### ISSUE THREAD

**Issue by vankeer** [cebor/rollup-plugin-angular#3]
I am trying to get rollup to work with a simple site using Angular RC4, however I get this error:

```
Error transforming C:\GIT\tests\ng2-site\node_modules\@angular\core\esm\src\animation\metadata.js with 'angular' plugin: ENOENT: no such file or directory, open 'C:\GIT\tests\ng2-site\node_modules\@angular\core\esm\src\animation\my-component-tpl.html'
Error: Error transforming C:\GIT\tests\ng2-site\node_modules\@angular\core\esm\src\animation\metadata.js with 'angular' plugin: ENOENT: no such file or directory, open 'C:\GIT\tests\ng2-site\node_modules\@angular\core\esm\src\animation\my-component-tpl.html'
    at Error (native)
    at Object.fs.openSync (fs.js:640:18)
    at Object.fs.readFileSync (fs.js:508:33)
    at C:\GIT\tests\ng2-site\node_modules\rollup-plugin-angular\dist\rollup-plugin-angular.js:16:19
    at RegExp.[Symbol.replace] (native)
    at String.replace (native)
    at insertText (C:\GIT\tests\ng2-site\node_modules\rollup-plugin-angular\dist\rollup-plugin-angular.js:15:13)
    at C:\GIT\tests\ng2-site\node_modules\rollup-plugin-angular\dist\rollup-plugin-angular.js:37:34
    at RegExp.[Symbol.replace] (native)
    at String.replace (native)
```

My rollup config looks like this:

```
import angular from 'rollup-plugin-angular';
import nodeResolve from 'rollup-plugin-node-resolve';
import typescript from 'rollup-plugin-typescript';

class RollupNG2 {
    constructor(options){
        this.options = options;
    }
    resolveId(id, from){
        if(id.startsWith("rxjs/")){
            return `${__dirname}/node_modules/rxjs-es/${id.replace("rxjs/", "")}.js`;
        }
    }
}
const rollupNG2 = (config) => new RollupNG2(config);

export default {
    entry: 'src/tmp/app/main.js',
    dest: 'src/tmp/app/bundle.js',

    format: 'iife',

    plugins: [
        rollupNG2(),
        angular(),
        nodeResolve({
            module: true,
            jsnext: true,
            main: true,
            skip: [],
            browser: true,
            extensions: ['.js'],
            preferBuiltins: false
        }),
        typescript()
    ]
};
```

And my entry file, compiled from TS to ES6:

```
import { bootstrap } from '@angular/platform-browser-dynamic';
import { enableProdMode } from '@angular/core';
import { disableDeprecatedForms, provideForms } from '@angular/forms';
import { AppComponent } from './app.component';
import { APP_ROUTER_PROVIDERS } from './app.routes';
import { SDK_SAMPLE_PROVIDERS } from 'ng2-library';
if (ENV === 'production') {
    enableProdMode();
}
bootstrap(AppComponent, [
    disableDeprecatedForms(),
    provideForms(),
    APP_ROUTER_PROVIDERS,
    SDK_SAMPLE_PROVIDERS
]);

//# sourceMappingURL=main.js.map
```

The only mentions of `my-component-tpl.html` I can find are in comments in @angular/core (within animation).

Any idea what's going on here? Thanks.

**Comment by cebor** [comment]
Hi, can you try to reorder rollup-plugins:

``` javascript
plugins: [   
        angular(),
        typescript(),
        rollupNG2(),
        nodeResolve({
            module: true,
            jsnext: true,
            main: true,
            skip: [],
            browser: true,
            extensions: ['.js'],
            preferBuiltins: false
        })   
    ]
```

**Comment by vankeer** [comment]
Hi @cebor, thanks for the quick reply. The order doesn't seem to make a difference though.

**Comment by jamesbirtles** [comment]
I just ended up excluding `@angular` from being processed

``` javascript
angular({
  exclude: [ 'node_modules/@angular/**' ],
})
```

**Comment by vankeer** [comment]
Yes, I guess the regex's of this plugin also match the @directive examples in the comments of the @angular source code.

**Comment by cebor** [comment]
Oh, i will dig into this tonight...

**Comment by cebor** [comment]
fixed in v0.2.7
